config/routes.rb

```diff
@@ -5,4 +5,7 @@
 
   devise_for :admin_users, ActiveAdmin::Devise.config
   ActiveAdmin.routes(self)
+
+  resources :orders, only: [:create, :index, :destroy]
```

Comment:
Not `#show`?

Reply:
See comment above.

Reply:
Okay

Reply:
What about the `:destroy` path?

Reply:
It is not included in this feature. It will be implemented in PT: https://www.pivotaltracker.com/story/show/150859781

Reply:
Now I understand. I will remove it.